public/modules/custom/grants_handler/src/Plugin/WebformHandler/GrantsHandler.php

```diff
@@ -502,6 +502,37 @@ public function alterForm(array &$form, FormStateInterface $form_state, WebformS
         ->addError('Application period is closed, no further editing is allowed.');
       $form['#disabled'] = TRUE;
     }
+
+    $all_current_errors = $this->grantsFormNavigationHelper->getAllErrors($webform_submission);
+    $storage = $form_state->getStorage();
```

Comment:
Ok, so the $storage could be used better here, to set our form errors to a subarray:

`
$storage = $form_state->getStorage();
$errors = $storage['errors']
...
 `